app/src/main/java/com/duckduckgo/app/global/DuckDuckGoApplication.kt

```diff
@@ -135,18 +92,10 @@ open class DuckDuckGoApplication : HasAndroidInjector, Application(), LifecycleO
             }
         }
 
-        loadTrackerData()
-        scheduleOfflinePixels()
-
-        notificationRegistrar.registerApp()
-        registerReceiver(shortcutReceiver, IntentFilter(ShortcutBuilder.SHORTCUT_ADDED_ACTION))
-
-        initializeHttpsUpgrader()
         submitUnsentFirePixels()
```

Comment:
I would need to do a bit more refactor here (no biggie) but prefer to do it in a follow-up PR